pkg/cmd/clientConfig.go

```diff
@@ -122,13 +122,18 @@ kubectl plugin mobile get clientconfig`,
 					ret = append(ret, svcConfig)
 				}
 			}
+			servicePinningHashes, err := retrieveHTTPSConfigForServices(ret)
```

Comment:
Can we make this skippable with a flag for the command. Should it actually be default behaviour I wonder. Either way a mechanism to make it optional seems reasonable